src/Symfony/Bundle/FrameworkBundle/Command/CacheClearCommand.php

```diff
@@ -45,7 +45,7 @@ class CacheClearCommand extends ContainerAwareCommand
     public function __construct($cacheClearer = null, Filesystem $filesystem = null)
     {
         if (!$cacheClearer instanceof CacheClearerInterface) {
-            @trigger_error(sprintf('Passing a command name as the first argument of "%s" is deprecated since version 3.4 and will be removed in 4.0. If the command was registered by convention, make it a service instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
The ` If the command was registered by convention, make it a service instead` hint is not useful here as we deprecated convention based registration meanwhile, which gives the very same.